### ISSUE TITLE
Unable to install notebooks to ZCU208 board

### ISSUE THREAD

**Issue by lnsrf** [strath-sdr/RFSoC-Book#5]
The instruction:  pip install https://github.com/strath-sdr/RFSoC-Book/archive/v1.1.0.tar.gz doesn't work.  It fails with "Failed to establish a new connection: [Errno -3] Temporary failure in name resolution."  It's presumably a DNS failure.  The nameserver is 127.0.0.53.  Doing a resolvectl status shows current scopes: none for Link4 (eth0).

My ZCU208 board is connected by ethernet to a Lenovo minidock providing USB to ethernet.  I set the pc static ip to 192.168.2.2.  I had previously not set the DNS server assignments (they were the default, auto I think.)  I tried manual setting and assigned IPv4 DNS servers 8.8.8.8 and 1.1.1.1, then rebooted the board.  It made no difference.

As a test, I installed PYNQ and Jupyter on the PC, was able to run the above pip command with no problem.  As a workaround, I can transfer the notebooks via samba, but that shouldn't be necessary.

I also did the same test using Binder, as suggested by someone on Stack Overflow.  Again, there was no problem with the notebook download (did not do the final install.)

Thank you.

**Comment by lnsrf** [comment]
This appears to be a network sharing issue since the board needs to share the wifi connection.  I tried setting up a network share, but windows 11 then said the IP address would be changed, so I'd lose the connection to the board.  I didn't see where I could enter the local network static address.

I also tried bridging the networks and then lost connectivity to both.

Thank you.

**Comment by lnsrf** [comment]
Resolved the problem.  I followed the instructions in https://byu-cpe.github.io/ecen427/documentation/setup-pynq-board/ to set up a share between eth0 and wifi.  Windows then displayed the LAN address it was going to use.  I then stopped the Jupyter server on the ZCU208 via the serial terminal and restarted it.  As the server started up it listed the dynamic eth0 address which I then entered in a browser, as addr:9090/lab.  It now works, and I was able to ping google.com.  I haven't tried pip install ..., but that will probably work.

Thanks.
